requirements-app.txt

```diff
@@ -15,7 +15,7 @@ gunicorn==20.0.4
 iso8601==0.1.13
 itsdangerous==1.1.0
 jsonschema==3.2.0
-marshmallow-sqlalchemy==0.23.1 # pyup: <0.24.1 # marshmallow v3 throws errors
+marshmallow-sqlalchemy==0.24.0 # pyup: <0.24.1 # marshmallow v3 throws errors
```

Comment:
blah, why doesnt this work :\

Reply:
Should be `<0.24.0` instead of `<0.24.1` perhaps?

Reply:
d'oh! yes you're absolutely right